lucet-runtime/lucet-runtime-internals/src/region/mmap.rs

```diff
@@ -50,6 +50,7 @@ use std::sync::{Arc, Mutex, Weak};
 /// 0xXXXX  --- global guard page ---
 /// 0xS000: +-----------------------| <-- Sigstack (at globals_start + globals_size + PAGE_SIZE)
 /// 0xSXXX: |  ......sigstack....   | // sigstack is SIGSTKSZ bytes
+/// 0xXXXX  -- sigstack guard page --
```

Comment:
Doesn't the guard page between the globals and the sigstack handle overflows of the signal stack?

Reply:
Yep! Reverted the sigstack guard page change.